src/scenes/home/benefit/benefit.css

```diff
@@ -31,6 +31,12 @@
   margin: 3vw;
 }
 
+.cf {
```

Comment:
Can we change this to be codeFellowsLogo to match the naming of the other class names?